### DISCUSSION TITLE
Simulate standard input

### DISCUSSION THREAD

**Discussion by AKuederle** [gitpython-developers/GitPython#1181] [Q&A] [answered]
Some git commands for example `git lfs smudge` can only read from STDIN. Is there a way to pass information to STDIN using the command wrapper of GitPython?

**Comment by Byron** [comment, marked as answer]
You should be able to [use the `istream` argument](https://github.com/gitpython-developers/GitPython/blob/b3778ec37d17a6eb781fa9c6b5e2009fa7542d77/git/cmd.py#L583:L583) of the `repo.git.execute(…)` method.

Even though I am closing this issue as question, please feel free to comment if this didn't work or if there are follow up questions.

**Reply by AKuederle** [reply]
I forgot to respond to the issue! Your suggestion worked. Thank you very much :)
